deeds-dapp-webapp/src/main/webapp/vue-app/dapp/common/components/EmailField.vue

```diff
@@ -148,7 +152,7 @@ export default {
     code() {
       if (this.code && Number.isInteger(Number(this.code)) && Number.isFinite(Number(this.code)) && this.code.length === 6) {
         this.sendingCode = true;
-        this.$authorizationCodeService.isCodeValid(this.code)
+        this.$authorizationCodeService.isCodeValid(this.code, this.email)
```

Comment:
Not needed, the email should be added to HttpSession.

```suggestion
        this.$authorizationCodeService.isCodeValid(this.code)
```